### ISSUE TITLE
Quick start additions and issue with config

### ISSUE THREAD

**Issue by mmccoo** [marksie1988/atomic-calendar-revive#384]
I'm fairly new to HA. I'm running a manual install of HA on raspi 4 Manjaro.
I've installed HACS and then Atomic revive

First, I recommend an addition to the quick start.

The quick start states: "If you use the Lovelace Editor then add the follwoing to Configuration -> Lovelace Dashboards -> Resources"

The resources option only appears if advanced mode is turned on in the user profile. The quick start should mention this.

Next, it is probably required to get authorization to the google calendar as described here:
https://www.home-assistant.io/integrations/calendar.google/

If so, the quick start should mention this. 

I'm not 100% sure on the authorization part as I haven't successfully gotten an Atomic Revive card to show due to the message:
this._config.entities is undefined

![image](https://user-images.githubusercontent.com/1066156/105472520-55426400-5c9c-11eb-8513-fde8a53d0176.png)

A bunch of googling has not yielded a way to get beyond this error.

:-)
Miles

## Checklist

- [ ] I searched the current issues and the issue is not known.
- [ ] I searched the troubleshooting and there's no fix available for this bug.
- [ ] I searched on Google, YouTube, Reddit and there's no fix.
- [ ] I am mostly sure that the problem is related to the code and isn't my fault.

**Comment by marksie1988** [comment]
The error that you are seeing is because you haven't added any calendar entities in the config. Unfortunately this needs to be done via the code editor.

**Comment by mmccoo** [comment]
ok. That helped.

I recommend adding to the quick start.

1) the advanced mode thing
2) looks like the google calendar oauth thing is a must have.
3) when mentioning "the config", I recommend being more specify. My initial thought was I had to do something in HA's config (since that's the most recent thing I needed to change for this). I believe you meant the config of the specific lovelace card one is trying to add.
4) perhaps more esoteric/noob. in the part about "entity: calendar.YOUR_CALENDAR_HERE", the your calender is the device_id as seen in the google_calendars.yaml file which magically applies after following the oauth instructions.

Either way, I'm unstuck. Thank you
Miles
